components/dial/stories/dial.stories.js

```diff
@@ -62,25 +63,71 @@ export default {
 	},
 };
 
+// the !auto-docs tag removes the component from the auto-template
 export const Default = Template.bind();
 Default.args = {};
+Default.tags = ['!autodocs'];
 
 export const DefaultExpress = Template.bind();
 DefaultExpress.args = {
 	express: true,
 };
+DefaultExpress.tags = ['!autodocs'];
 
 export const Small = Template.bind();
 Small.args = {
 	size: "s",
 };
+Small.tags = ['!autodocs'];
 
 export const WithLabel = Template.bind();
 WithLabel.args = {
 	label: "Volume",
 };
+WithLabel.tags = ['!autodocs'];
 
 export const Disabled = Template.bind();
 Disabled.args = {
 	isDisabled: true,
 };
+Disabled.tags = ['!autodocs'];
+
+/*
+ * Stories for the MDX "docs" only.
+ * "storyName" refers to the display name/heading for a component
+ */
+export const DialGroup = (args) => html `
+<div style="display: grid; grid-template-columns: 48px 48px; gap: 16px;">
+	${Template({
+		...args,
+	})}
+	${Template({
+		...args,
+		isDisabled: true,
+	})}
+	${Template({
+		...args,
+		size: "s",
+	})}
+	${Template({
+		...args,
+		size: "s",
+		isDisabled: true,
+	})}
+	</div>
+`;
+DialGroup.tags = ["docs-only"];
+DialGroup.storyName = "Standard";
```

Comment:
I also found `storyName` here: https://storybook.js.org/docs/writing-stories/naming-components-and-hierarchy#single-story-hoisting so I could override the component name  when it gets rendered.